### PR TITLE
feat(component): material navbar is improved

### PR DESCRIPTION
**Pull request by ggarg2**
- MdTabLinks are removed from nav bar links. Now using md-button.
- Removed unnecessary tag from app component
- Introduced router link .active css

### CODE REVIEW

**Comment by patrickmichalina** [comment]
@ggarg2 thanks for this. I like the new navbar, looks much better. Can you fix that failing test then I can merge it.

File: src/client/app/shared/navbar/navbar.component.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            width: 96px;
+        }
+        box-shadow: 0 3px 5px -1px rgba(0, 0, 0, .2),
+        0 6px 10px 0 rgba(0, 0, 0, .14),
+        0 1px 18px 0 rgba(0, 0, 0, .12);
+        .active {
+            box-shadow: 4px 3px 8px 2px #26588b;
+        }
+    }
+    .app-navbar {
```
Comment (patrickmichalina) [inline src/client/app/shared/navbar/navbar.component.scss]: Can you move this content into the `nav` element and get rid of the class `.app-navbar`?

File: src/client/app/shared/navbar/navbar.component.html [hunk 2]
```diff
@@ -1,48 +1,11 @@
...
-    .github-corner:hover .octo-arm {
-      animation: none
-    }
-    .github-corner .octo-arm {
-      animation: octocat-wave 560ms ease-in-out
-    }
-  }
-</style>
\ No newline at end of file
+<nav class="app-navbar">
```
Comment (patrickmichalina) [inline src/client/app/shared/navbar/navbar.component.html]: not sure this class is needed

**Review by patrickmichalina — CHANGES_REQUESTED** [review]

File: src/client/app/shared/navbar/navbar.component.scss (hunk 1 above)
Comment (ggarg2) [reply src/client/app/shared/navbar/navbar.component.scss]: Hey Sorry, Just saw your message. Yes we can move the content of .app-navbar into nav

File: src/client/app/shared/navbar/navbar.component.html (hunk 2 above)
Comment (ggarg2) [reply src/client/app/shared/navbar/navbar.component.html]: we can remove it. I just created for css example.

**Comment by ggarg2** [comment]
Hi,
one question, how are you creating snapshots? 
I tried but i got failed

**Comment by patrickmichalina** [comment]
@ggarg2 please run `npm run test.update` in order to confirm snapshots should be updated

**Comment by ggarg2** [comment]
awesome it worked

**Comment by ggarg2** [comment]
changes are done. i have committed all changes in fork repository but not able to create new pull request

Please help

**Comment by patrickmichalina** [comment]
@ggarg2 same pull request, just have to push the changes which it appears you did

**Review by patrickmichalina — APPROVED** [review]

**Comment by ggarg2** [comment]
Oh okay.

**Pull request merged**